codemod_tox/envlist.py

```diff
@@ -27,6 +28,31 @@ def __iter__(self) -> Generator[str, None, None]:
         for e in self.envs:
             yield from iter(e)
 
+    def changefirst(
+        self,
+        predicate: Callable[[ToxEnv], bool],
+        mapper: Callable[[ToxEnv], Optional[ToxEnv]],
+    ) -> "ToxEnvlist":
```

Comment:
Why is this called `changefirst`? It's a bit unintuitive. 

modify_if_exists() ?